dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServerTest.java

```diff
@@ -109,7 +107,7 @@ public void testSimpleServlet() throws Exception {
      *                       if something goes horribly wrong
      */
     @Test
-    @SkipForRepeat({ EE9_FEATURES, EE10_FEATURES })
+    @SkipForRepeat({ EE9_FEATURES, EE10_FEATURES, EE11_FEATURES })
```

Comment:
Are there plans to improve SkipForRepeat behavior? It would be nice if we didn't need to update the skips for every release. If the repeat code treated a particular EE version as the minimum, then it could be skipped for EE9 and anything higher.

Reply:
I can create an issue?

Reply:
Sure go ahead and open an issue.

Reply:
Jared may be a solution.